pkgs/top-level/aliases.nix

```
@@ -613,8 +615,10 @@ mapAliases ({
   nixopsUnstable = nixops_unstable; # Added 2022-03-03
   nixosTest = testers.nixosTest; # Added 2022-05-05
   nmap-unfree = nmap; # Added 2021-04-06
+  nodejs_14 = throw "nodejs_14 has been removed as it is EOL."; # Added 2023-30-10
```

looks like ofborg is failing because of this, we might also need to same thing for nodejs 16

```suggestion
  nodejs_14 = throw "nodejs_14 has been removed as it is EOL."; # Added 2023-30-10
  nodejs-slim_14 = throw "nodejs-slim_14 has been removed as it is EOL."; # Added 2023-30-10
```